spec/person_spec.rb

```diff
@@ -2,19 +2,45 @@
 require_relative '../person'
 
 RSpec.describe Person do
-  describe '#of_age?' do
-    context 'when the person is 18 years old or older' do
-      it 'returns true' do
-        person = Person.new(18)
-        expect(person.send(:of_age?)).to be true
+    describe '#can_use_services?' do
+      context 'when the person is of age' do
+        it 'returns true' do
+          person = Person.new(20)
```

Comment:
- [Optional] Good job adding the test for the missing methods in the person class as suggested earlier. But I notice you have not tested some of the classes at all. this includes the `BaseDecorator`  and others. Please try to cover all the methods in all classes.